assets/src/edit-story/app/media/media3p/attribution.js

```diff
@@ -103,20 +116,26 @@ export function CoverrAttribution() {
         <Text
           forwardedAs="span"
           size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+          aria-label={getAriaLabel(MEDIA_PROVIDER.coverr)}
         >
           {__('Powered by', 'web-stories')}
         </Text>
-        <CoverrLogo />
+        <CoverrLogo aria-hidden />
       </AttributionPill>
     </a>
   );
 }
 
 export function TenorAttribution() {
   return (
-    <a href={tenorUrl} target={'_blank'} rel={'noreferrer'}>
+    <a
+      href={tenorUrl}
+      target={'_blank'}
+      rel={'noreferrer'}
+      aria-label={getAriaLabel(MEDIA_PROVIDER.tenor)}
+    >
       <AttributionPill>
```

Comment:
I don't understand why tenor is different here? Why can't we keep it the same. Text "Powered by" and a tenor logo. The svg logo should have some kind of alt tag.

Reply:
The Tenor logo is the only one that contains the text "powered by" in the image itself. See https://github.com/google/web-stories-wp/blob/7eb02795c5349585ea5083e382f924eb1f9e0188/assets/src/edit-story/icons/tenor_logo_white.svg

So there's no need for separate "Powered by" text.

But you're right, I think we should be able to set the SVG's `<title>` by doing

```jsx
<TenorLogo title={getAriaLabel(MEDIA_PROVIDER.tenor)} />
```

No need for `aria-label` and `aria-hidden` in this case.

But needs testing to confirm.

Reply:
This is more a case of uniformity. Why is this different? Why does it need to be different?

Reply:
That's the media3p provider attribution logo we were originally provided by Tenor. We can't just simply change it, so we have to stick with it.

Reply:
Why can't it be changed?

Reply:
Because this specific attribution is part of the terms of using that provider. I can reach out to see if it's OK to use another SVG and if they can provide that, but seems like way too much effort for this minor inconsistency.

Reply:
I guess I'm looking at this in a different way, just giving the anchor tag the info it needs without needing to know anything about the SVG at all. It announces better and feels more straight forward. Also allows implementation to be uniform in the component.

Reply:
Works for me. Let's keep this as-is.

Anything beyond that is out of scope for this fix.